frontend/pages/passport-application/index.tsx

```diff
@@ -1,3 +1,106 @@
-export default function PassportApplication() {
-  return <div className="passportApplication">I AM PASSPORT!!!!!!</div>
-}
\ No newline at end of file
+import Link from "next/link"
+
+import Logo from "../../shared-components/logo"
+import styles from "./passport.module.css"
+import { USStateSelect } from "../../shared-components/state-select/UsStateSelect"
+
+function handleSubmit() {
+  // TODO: add form submission logic
+}
+
+export default function Passport({ name = "Herbert Q. Placeholder" }) {
+  const {
+    ppWrapper,
+    cityInput,
+    logoWrapper,
+    inputWrapper,
+    ppForm,
+    ppHeading,
+    ppParaWrapper,
+    ppPara,
+    ppRedirect,
+    ppTextarea,
+    ppButton,
+    ppAddress,
+    ppAddInput,
+    ppZip,
+    ppState,
+    ppCity,
+    zipInput,
+    ppLabel
+  } = styles
+  return (
+    <div className={ppWrapper}>
+      <div className={logoWrapper}>
+        <Logo />
+      </div>
+      <h1 className={ppHeading}>Passport Account Application</h1>
+      <div className={ppParaWrapper}>
+        <p className={ppPara}>
+          Hello <em>{name}</em>, thank you for your continued interest in the National Police Data
+          Coalition.
+        </p>
+        <p className={ppPara}>
+          We are able to provide access to legally protected data to users with the appropriate
+          permissions. This form will submit your profile for approval.
+        </p>
+      </div>
+      <form className={ppForm}>
+        <div className={[ppAddress, inputWrapper].join(" ")}>
+          <label htmlFor="address" className={styles.label}>
+            <strong>Street Address:</strong>
+          </label>
+          <input
+            type="text"
+            id="address"
+            required={true}
```

Comment:
This differs fairly significantly from the error-validation patterns used in Login & ViewerRegistration - BUT imho the error validation stuff felt like it scope-crept wayyy too far out of a reasonable size for these mainly-UI-based tickets

Definitely not a blocker but I think we might want to create a follow-up ticket to work on error validation between the tree "enrollment" type pages!